arrow-cast/src/display.rs

```diff
@@ -72,6 +72,9 @@ pub struct FormatOptions<'a> {
     time_format: TimeFormat<'a>,
     /// Duration format
     duration_format: DurationFormat,
+    /// If set to `true`, decimal values will be formatted as strings instead of numbers
+    /// This is useful for JSON output to preserve precision
```

Comment:
It could be ignored, but I see that two close sentences usually end up with dots.
```suggestion
    /// If set to `true`, decimal values will be formatted as strings instead of numbers.
    /// This is useful for JSON output to preserve precision.
```